packages/wonder-blocks-dropdown/components/dropdown-core.test.js

```diff
@@ -485,4 +486,83 @@ describe("DropdownCore", () => {
         option1.simulate("click");
         expect(onClick1).toHaveBeenCalledTimes(1);
     });
+
+    it("shows SearchTextInput when onSearchTextChanged and searchText is provided", () => {
+        // Arrange, Act
```

Comment:
Can you change this to distinct sections for `// Arrange` and `// Act` please.

The structure is helpful for future maintenance as it encourages folks to put things in the right place. In this example, the `handleSearchTextChanged` is clearly the `// Arrange` and the `setProps` is the `// Act`.

Same for other test cases.